src/imap.rs

```diff
@@ -1032,11 +968,7 @@ impl Imap {
 
         // Server does not support MOVE or MOVE failed.
         // Copy the message to the destination folder and mark the record for deletion.
-        let session = self
-            .session
-            .as_mut()
-            .context("no session while attempting to COPY messages")?;
-        match session.uid_copy(&set, &target).await {
+        match self.uid_copy(&set, &target).await {
```

Comment:
Could u point me to where messages are deleted after copying? Saw only uid_copy() here...